### PR TITLE
added trackResize to default options map

### PR DESCRIPTION
**Pull request by behboud**
see #62 .

my first contribution, so please advice if anything missing.

### CODE REVIEW

File: src/services/leafletMapDefaults.js [hunk 1]
```diff
@@ -101,69 +102,8 @@ angular.module('ui-leaflet').factory('leafletMapDefaults', function ($q, leaflet
         setDefaults: function (userDefaults, scopeId) {
             var newDefaults = _getDefaults();
 
-            if (isDefined(userDefaults)) {
```
Comment (nmccready) [inline src/services/leafletMapDefaults.js]: This does seem way less complicated and I am not really sure the reasoning behind the way it was. @elesdoar any thoughts?

Comment (elesdoar) [inline src/services/leafletMapDefaults.js]: Angular extends don't work in complex objects, or object with functions, this could generate errors.

Comment (behboud) [inline src/services/leafletMapDefaults.js]: how about using angular.merge? https://docs.angularjs.org/api/ng/function/angular.merge
seems to be more safe with complex objects.

Comment (nmccready) [inline src/services/leafletMapDefaults.js]: Yes merge is for complex, but it will limit us to angular 1.4, which I am fine w/

**Comment by nmccready** [comment]
@behboud why close?

**Comment by behboud** [comment]
this is all new to me. thought I need to start a new pull request. also: angular.merge causes errors. so I need to revert and add trackResize as property

**Comment by nmccready** [comment]
k

**Pull request closed without merging**